### PR TITLE
update points and series flask endpoints to filter by facet id

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: server/routes/shared_api/observation/point.py [hunk 1]
```diff
@@ -69,8 +69,10 @@ def point_within():
   if not variables:
     return 'error: must provide a `variables` field', 400
   date = request.args.get('date') or 'LATEST'
+  facet_id = request.args.get('facetId') or None
```
Comment (shifucun) [inline server/routes/shared_api/observation/point.py:72]: shall we use "facetIds" here as well for consistency?

File: server/routes/shared_api/observation/series.py [hunk 2]
```diff
@@ -60,7 +60,8 @@ def series_all():
     return 'error: must provide a `entities` field', 400
   if not variables:
     return 'error: must provide a `variables` field', 400
-  return fetch.series_core(entities, variables, True)
+  facet_ids = list(filter(lambda x: x != "", request.args.getlist('facetIds')))
+  return fetch.series_core(entities, variables, True, facet_ids)
```
Comment (shifucun) [inline server/routes/shared_api/observation/series.py:64]: also add after line 49?

File: server/routes/shared_api/observation/point.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/shared_api/observation/point.py:72]: since this is only a single facetId, I think facetId makes more sense? wdyt?

Comment (shifucun) [reply server/routes/shared_api/observation/point.py:72]: Can you remind me what does a list of facet id filter implies? Say i want to fetch [facet1, facet2], but some data has either facet1 or facet2, would that work? 

The result here does return 1 facet, but can the filter be  multiple so I can ask for either 
Census or WorldBank data, whichever has data.

**Review by shifucun — APPROVED** [review]
thanks for update!

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**